Makefile

```diff
@@ -109,11 +109,12 @@ generate:
 .push-build-image:
 	$(DOCKER) push $(BUILDIMAGE)
 
-$(DOCKER_TARGETS): docker-%: .build-image
-	@echo "Running 'make $(*)' in docker container $(BUILDIMAGE)"
+$(DOCKER_TARGETS): docker-%: 
+	@echo "Running 'make $(*)' in container image $(BUILDIMAGE)"
 	$(DOCKER) run \
 		--rm \
-		-e GOCACHE=/tmp/.cache \
+		-e GOCACHE=/tmp/.cache/go \
+		-e GOMODCACHE=/tmp/.cache/gopath \
```

Comment:
nit: should this be `/tmp/.cache/go/mod` or something ... `gopath` and `GOMODCACHE` seem unrelated